go/vt/vtgate/vtgate_test.go

```diff
@@ -2709,3 +2710,31 @@ func TestParseKeyspaceOptionalShard(t *testing.T) {
 		}
 	}
 }
+
+func TestErrorTruncation(t *testing.T) {
+	*sqlparser.TruncateErrLen = 30
+
+	query := map[string]interface{}{
```

Comment:
Like in the other comment, you can build a `wantVars` variable and perform an equality comparison using `reflect.DeepEqual`.